<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyManagement</class>
 <widget class="QMainWindow" name="PropertyManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Property Management</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="addPropertyPushButton">
          <property name="text">
           <string>Add Property...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removePropertyPushButton">
          <property name="text">
           <string>Remove Property...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="renamePropertyPushButton">
          <property name="text">
           <string>Rename Property...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updatePropertyPushButton">
          <property name="text">
           <string>Update Property...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="changePropertyDataPushButton">
          <property name="text">
           <string>Change Property Data...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="quitPushButton">
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
